runtime/src/accounts_index.rs

```diff
@@ -1685,11 +1685,14 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> AccountsIndex<T, U> {
     }
 
     /// return Vec<Vec<>> because the internal vecs are already allocated per bin
-    pub fn retrieve_duplicate_keys_from_startup(&self) -> Vec<Vec<(Slot, Pubkey)>> {
+    pub(crate) fn populate_and_retrieve_duplicate_keys_from_startup(
+        &self,
+    ) -> Vec<Vec<(Slot, Pubkey)>> {
         (0..self.bins())
+            .into_par_iter()
```

Comment:
`populate_and_retrieve_duplicate_keys_from_startup` now does more work than it used to. It used to just return the vec it had previously stored. Now it loads entries from disk buckets and populates entries in the in-mem hashmap per bucket. Since there are currently 8k buckets and there could be much more, it makes sense to do this in parallel. I did not measure whether it made a difference. Parallelizing over buckets seems like a good idea. And, it is at startup where there is nothing else going on while we do this.